crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -629,6 +629,166 @@ impl Torus {
     }
 }
 
+/// A 3D triangle primitive.
+#[derive(Clone, Copy, Debug, PartialEq)]
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+pub struct Triangle3d {
+    /// The vertices of the triangle
+    pub vertices: [Vec3; 3],
+}
+
+/// An error that occurs when a triangle is degenerate, meaning it has zero area.
+/// This can happen when the three vertices are collinear or nearly collinear.
+#[derive(Debug, Clone)]
+pub struct DegenerateTriangleError;
+
+impl std::fmt::Display for DegenerateTriangleError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Triangle is degenerate")
+    }
+}
+
+impl Primitive3d for Triangle3d {}
+
+impl Default for Triangle3d {
+    /// Returns the default [`Triangle3d`] with vertices at the `[-0.5, 0.0, 0.0]`, `[0.5, 0.0, 0.0]`, and `[0.0, 0.5, 0.0]`
+    fn default() -> Self {
+        Self {
+            vertices: [
+                Vec3::new(-0.5, 0.0, 0.0),
+                Vec3::new(0.5, 0.0, 0.0),
+                Vec3::new(0.0, 0.5, 0.0),
+            ],
+        }
+    }
```

Comment:
This is not consistent with `Triangle2d` or any other primitives. The bounding box for this is a 1 x 0.5 rectangle offset to be above the origin. It should be a 1 x 1 rectangle centered on the origin. `Triangle2d` also has the top vertex first (although that is kind of arbitrary).
```suggestion
    /// Returns the default [`Triangle3d`] with vertices at the `[0.0, 0.5, 0.0]`, `[-0.5, -0.5, 0.0]`, and `[0.5, -0.5, 0.0]`
    fn default() -> Self {
        Self {
            vertices: [
                Vec3::new(0.0, 0.5, 0.0),
                Vec3::new(-0.5, -0.5, 0.0),
                Vec3::new(0.5, -0.5, 0.0),
            ],
        }
    }
```